PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Assignee Section
+
+## Description
+[FILL]
+
+Closes Issue : [#FILL]
```

Comment:
I think this syntax is wrong. Simplest would be just `closes #24` so maybe write `closes #<add your issue number here>`
Refrence [here](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) reference in.`